project/Build.scala

```diff
@@ -110,6 +110,25 @@ object Build {
     if (condition) List(testDir)
     else Nil
 
+  private def addScalaJSCompilerOption(option: String): Setting[_] =
+    addScalaJSCompilerOption(Def.setting(option))
+
+  private def addScalaJSCompilerOption(option: Def.Initialize[String]): Setting[_] =
+    addScalaJSCompilerOption(None, option)
+
+  private def addScalaJSCompilerOptionInConfig(config: Configuration,
+      option: String): Setting[_] = {
+    addScalaJSCompilerOption(Some(config), Def.setting(option))
+  }
```

Comment:
Can't we use sbt's `inConfig` for this?

Reply:
No, we can't, because that would also apply the config to the `option`. That's only really a problem for the case with the `Initialize[String]`, but then I thought I would keep consistency.

(Also, `inConfig` always returns a `Seq[Setting[_]]`, and at the existing call sites I needed a single `Setting[_]`, so using `inConfig` would have required more refactoring.)